pulp_rpm/tests/functional/api/test_publish.py

```diff
@@ -417,20 +423,24 @@ def get_checksum_types(self, **kwargs):
 
         return repomd_checksum_types, primary_checksum_types
 
-    def test_unspecified_checksum_types(self):
-        """Sync and publish an RPM repository and verify the checksum."""
-        repomd_checksum_types, primary_checksum_types = self.get_checksum_types()
+    def test_on_demand_unspecified_checksum_types(self):
+        """Sync and publish an RPM repository and verify the checksum types."""
+        repomd_checksum_types, primary_checksum_types = self.get_checksum_types(
+            fixture_url=RPM_SHA512_FIXTURE_URL, policy="on_demand"
+        )
 
         for repomd_type, repomd_checksum_type in repomd_checksum_types.items():
             self.assertEqual(repomd_checksum_type, "sha256")
 
         for package, package_checksum_type in primary_checksum_types.items():
-            self.assertEqual(package_checksum_type, "sha256")
+            # since none of the packages in question have sha384 checksums, the
```

Comment:
copy-paste issue? not sure how sha384 got here.

Reply:
Deliberately.  I wanted to throw in some checksums that aren't the default choices to make sure it's working

Reply:
Ignore my previous comment.  Yes, it was a mistake.  I was viewing on a mobile app this morning and mixed up the various places.